### PR TITLE
feat(currentSprintButton):added sprint button and its functionality

### PR DESCRIPTION
**Pull request by Joelzac537**
### Functionality:
Shows the current sprint.

### Solution:
To make the button align with backlog button

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
By clicking the button it will take you to the current sprint.

### CODE REVIEW

**Review by bighnesh0404 — APPROVED** [review]
Code has been reviewed and its working properly and ready to merge

File: worktrolly-ui/src/app/body/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -105,6 +105,11 @@ export class DashboardComponent implements OnInit {
     this.currentSprintName = "Backlog";
   }
 
+  currentSprint(){
+    this.getCurrentSprint();
+    this.currentSprintName = "S" + this.currentSprintNumber;
```
Comment (viv9k) [inline worktrolly-ui/src/app/body/dashboard/dashboard.component.ts:110]: I think name is already being set in line 77, it can be removed from here.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please update it asap

File: worktrolly-ui/src/app/body/dashboard/dashboard.component.ts (hunk 1 above)
Comment (Joelzac537) [reply worktrolly-ui/src/app/body/dashboard/dashboard.component.ts]: updated

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**